content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -218,12 +218,36 @@ equal to one. The `nginx` container is granted 2 exclusive CPUs.
 
 #### Static policy options
 
+The following policy options exist for the static `CPUManager` policy:
```

Comment:
I think we should clearly indicate here which of the following options are visible by default and the ones that are not and how that can be achieved by using feature gates. The feature gates part is covered in https://github.com/kubernetes/website/pull/29709 but I think we should add a subsection:
 `##### Alpha options (hidden by default)` and move the content corresponding to `distribute-cpus-across-numa` that we have here in that subsection.

Reply:
Instead of having a whole subsection, for each, I think it might be sufficient (if not better) to just list these as:
>
> * `full-pcpus-only` (beta, visible by default)
> * `distribute-cpus-across-numa` (alpha, hidden by default)
>

Reply:
Sure, we only have two options at the moment so doesn't make much difference. When more options are introduced and the list grows we might want to better represent this information so we can clearly see all the options that are visible by default and the ones that are hidden.